src/modules/tools/zprobe/DeltaGridStrategy.cpp

```diff
@@ -0,0 +1,600 @@
+/*
+ This code is derived from (and mostly copied from) Johann Rocholls code at https://github.com/jcrocholl/Marlin/blob/deltabot/Marlin/Marlin_main.cpp
+ license is the same as his code.
```

Comment:
Ah, I just used the Eclipse create .cpp and .h function, I will copy the Smoothie license into each of the files.